src/garage/torch/modules/gaussian_mlp_module.py

```diff
@@ -0,0 +1,231 @@
+import torch
+from torch import nn
+from torch.distributions import MultivariateNormal
+
+from garage.torch.modules import MLPModule
+
+
+class GaussianMLPModule(nn.Module):
+    """
+    GaussianMLPModel.
+
+    Args:
+        input_dim (int): Input dimension of the model.
+        output_dim (int): Output dimension of the model.
+        hidden_sizes (list[int]): Output dimension of dense layer(s) for
+            the MLP for mean. For example, (32, 32) means the MLP consists
+            of two hidden layers, each with 32 hidden units.
+        hidden_nonlinearity (callable): Activation function for intermediate
+            dense layer(s). It should return a torch.Tensor. Set it to
+            None to maintain a linear activation.
+        hidden_w_init (callable): Initializer function for the weight
+            of intermediate dense layer(s). The function should return a
+            torch.Tensor.
+        hidden_b_init (callable): Initializer function for the bias
+            of intermediate dense layer(s). The function should return a
+            torch.Tensor.
+        output_nonlinearity (callable): Activation function for output dense
+            layer. It should return a torch.Tensor. Set it to None to
+            maintain a linear activation.
+        output_w_init (callable): Initializer function for the weight
+            of output dense layer(s). The function should return a
+            torch.Tensor.
+        output_b_init (callable): Initializer function for the bias
+            of output dense layer(s). The function should return a
+            torch.Tensor.
+        learn_std (bool): Is std trainable.
+        init_std (float): Initial value for std.
+            > std_parameterization
+                'exp' : plain value (not log or exponentiated)
```

Comment:
this parameter should accept an unparameterized value for std, and convert to the appropriate parameterization.

Reply:
You mean this module have to accept only plain std value?

Reply:
@yonghyuc if this is true, it is a bug. @ahtsan can you check?

if the init_std/min_std/max_std are being processed wrong in TF, please file an issue and assign it to Anson. It should be easy to fix.

Reply:
@ryanjulian @yonghyuc We talked and agreed that it is true and it is a bug. I will file an issue about it.

Reply:
ok. please file the issue. preferably fix before release if it's not huge.

Reply:
fixed